benchmarks/span/util.py

```diff
@@ -0,0 +1,27 @@
+from random import choice
+from random import randint
+import string
+
+
+def gen_tags(ntags=50, ltags=30):
+    tag_values = [rands(size=ltags) for _ in range(ntags)]
+    tag_keys = [rands(size=ltags) for _ in range(ntags)]
+    tags = [tag_keys, tag_values]
```

Comment:
Might make it easier to return a list of tuples `[(k,v),...]`.

Reply:
I changed it to a dictionary so I can use set_tags instead of a for loop + set_tag